### ISSUE TITLE
[Data] Automate schedule downloads

### ISSUE THREAD

**Issue by lauriemerrell** [chihacknight/chn-ghost-buses#18]

**Comment by mrscraps13** [comment]
wanted feed back on this, please let me know!
 @lauriemerrell @KyleDolezal 
"""
with open("infile", "rb") as in_file, open("out-file", "wb") as out_file:
    chunk = in_file.read(chunk_size)

    if chunk == b"":
        break

    out_file.write(chunk)
"""

**Comment by KyleDolezal** [comment]
@mrscraps13 It looks good to me. I can see similar working examples, such as [here](https://www.programcreek.com/python/?code=cheshirekow%2Fcmake_format%2Fcmake_format-master%2Ftangent%2Ftooling%2Fexport_sparse_subrepo.py). Is this code part of a branch? I'm wondering if I could see it in context.

**Comment by lauriemerrell** [comment]
Agree with @KyleDolezal -- I think that in my day job where we download feeds, we just use `requests` and basically `request.get(<SCHEDULE_URL>)` and then just save the response content. Here's an example: https://github.com/cal-itp/data-infra/blob/main/airflow/dags/gtfs_downloader/download_data.py#L35-L78, it's a bit hard to follow because there's some other config stuff going on but maybe helpful?

**Comment by mrscraps13** [comment]
im a bit lost about the 'context' piece. the way i thought about this was reading the file by chunks. could someone provide a bit more guidance :)
